database/clickhouse/README.md

```diff
@@ -1,13 +1,14 @@
 # ClickHouse
 
-`clickhouse://host:port?username=user&password=qwerty&database=clicks&x-multi-statement=true`
+`clickhouse://host:port?username=user&password=qwerty&database=clicks&x-multi-statement=true&x-table-engine=MergeTree`
```

Comment:
The example shouldn't encourage the use of MergeTree as the [log family](https://clickhouse.tech/docs/en/engines/table-engines/log-family/) supports the migrations table needs best.

But we should add a note to recommend the usage of MergeTree with that specific backup tool

Reply:
Changed to make the use cases clear